### DISCUSSION TITLE
Possible mistake in guide about clipboard plugin

### DISCUSSION THREAD

**Discussion by StepanZagray** [tauri-apps/tauri#10290] [Q&A]
Here are [docs](https://v2.tauri.app/plugin/clipboard/).
Code from guide:
```Rust
use tauri_plugin_clipboard_manager::ClipboardExt;

// Write content to clipboard
let clipboard_content = tauri_plugin_clipboard_manager::ClipKind::PlainText {
    label: Some("Label".to_string()),
    text: "Tauri is awesome!".to_string(),
};
app.clipboard().write(clipboard_content).unwrap();

// Read content from clipboard
let content = app.clipboard().read();
println!("{:?}", content.unwrap());
// Prints "Tauri is awesome!" to the terminal
```

The question is, where are `app` and `ClipKind` appears from? 
`ClipKind` does not exists in that path
And why we importing `use tauri_plugin_clipboard_manager::ClipboardExt;` when we are not using anywhere?

**Comment by StepanZagray** [comment]
I also noticed that `app` appears in other [guides](https://v2.tauri.app/plugin/dialog/#build-a-file-selector-dialog) without any context

**Reply by FabianLars** [reply]
`app` is supposed to be an AppHandle instance which you have access to in the `setup` hook or in commands via https://v2.tauri.app/develop/calling-rust/#accessing-an-apphandle-in-commands

I will address the rest of your initial comment in https://github.com/tauri-apps/tauri-docs/issues/2412
